float32[] xyz_clouds
int32[] l_clouds
---
visualization_msgs/MarkerArray output
